### PR TITLE
Add EnergyWeb article

### PR DESCRIPTION
**Pull request by wtzb**

### CODE REVIEW

File: how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+title: How To Access Your Energy Web Chain Tokens
+description: Networks & Nodes
+tags:
+  - Networks
+priority: 20
+date_published: '2020-09-22'
+date_modified: '2020-09-22'
+---
+
+Energy Web Chain ("EWC") is a blockchain designed to be used for the energy sector. Its utility token is Energy Web Token ("EWT"), and this article will explain how you can access these tokens using MyCrypto.
```
Comment (LuckyLuker) [inline how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md:11]: no comma after ("EWT")

File: how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+EWC is a separate network and blockchain from Ethereum, and is not a token on the Ethereum blockchain. The term "token" may cause confusion, but keep in mind that EWT is separate from Ethereum.
```
Comment (LuckyLuker) [inline how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md:13]: No comma after "EWC is a separate network and blockchain from Ethereum"

File: how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Keep in mind that when you are connected to this network, you will not see your Ethereum or ERC-20 funds. Simply switch back to the Ethereum network if you wish to access these funds again.
+
+## Accessing your Energy Web Tokens
+
+<Accordion>
+<AccordionItem title="Ledger">
+
+To interact with EWT using your Ledger device, you have to use the EnergyWebChain app on your Ledger device, **not** the Ethereum app. If you do not have this app installed, you can use [Ledger Live](https://www.ledger.com/ledger-live) to install it.
+
+After accessing your Ledger device with MyCrypto, you will notice that a list of different addresses will show up, instead of the addresses that you would see if you were connected to the Ethereum network. You are free to use these for your Energy Web Tokens.
```
Comment (LuckyLuker) [inline how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md:30]: No comma after "list of different addresses will show up"

File: how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+If you have accidentally sent EWT to your Ethereum addresses previously, you can still access these by clicking the drop-down field next to "Addresses", and selecting the "Ledger (ETH)" option.
```
Comment (LuckyLuker) [inline how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md:32]: No comma after "Addresses"

File: how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+![Ethereum addresses on Ledger](../../assets/how-to/nodes-networks/how-to-access-energy-web-chain-tokens/ledger-derivation.png)
+
+You will see that your Ethereum addresses will show up, together with its EWT funds.
```
Comment (LuckyLuker) [inline how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md:36]: Recommended change: "You will see your Ethereum addresses displayed alongside their EWT funds."

File: how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+</AccordionItem>
+<AccordionItem title="Trezor">
+
+Access your Trezor device like usual. You will notice that a list of different addresses will show up, instead of the addresses that you would see if you were connected to the Ethereum network. You are free to use these for your Energy Web Tokens.
+
+If you have accidentally sent EWT to your Ethereum addresses previously, you can still access these by clicking the drop-down field next to "Addresses", and selecting the "TREZOR (ETH)" option.
```
Comment (LuckyLuker) [inline how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md:43]: No comma after "Addresses"

File: how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+![Ethereum addresses on Trezor](../../assets/how-to/nodes-networks/how-to-access-energy-web-chain-tokens/trezor-derivation.png)
+
+You will see that your Ethereum addresses will show up, together with its EWT funds.
```
Comment (LuckyLuker) [inline how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md:47]: Recommended change: "You will see your Ethereum addresses displayed alongside their EWT funds."

File: how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+</AccordionItem>
+<AccordionItem title="Private Key/Mnemonic Phrase/Keystore File">
+
+You can access your address like usual, and you'll be able to see your EWT right away.
```
Comment (LuckyLuker) [inline how-to/nodes-networks/how-to-access-energy-web-chain-tokens.md:52]: No comma after "usual"

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**